apps/condo/domains/settings/access/MobileFeatureConfig.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Generated by `createschema settings.MobileFeatureConfig 'organization:Relationship:Organization:CASCADE; emergencyPhone:Text; commonPhone:Text; onlyGreaterThanPreviousMeterReadingIsEnabled:Checkbox; meta:Json; ticketSubmittingIsEnabled:Checkbox'`
+ */
+
+const { uniq, map, get } = require('lodash')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { find, getById } = require('@open-condo/keystone/schema')
+
+const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
+const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canReadMobileFeatureConfigs ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    if (user.isAdmin || user.isSupport) return {}
+
+    if (user.type === RESIDENT) {
+        const residents = await find('Resident', { user: { id: user.id } })
+        const organizations = uniq(map(residents, 'organization'))
+
+        if (residents.length > 0) {
+            return {
+                organization: { id_in: organizations },
+                deletedAt: null,
+            }
+        }
+        return false
+    }
+
+    return {
+        organization: {
+            OR: [
+                queryOrganizationEmployeeFor(user.id),
+                queryOrganizationEmployeeFromRelatedOrganizationFor(user.id),
+            ],
+        },
+    }
+}
+
+async function canManageMobileFeatureConfigs (attrs) {
+    const { authentication: { item: user }, originalInput, operation, itemId } = attrs
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.type === RESIDENT) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    let organizationId
+    if (operation === 'create') {
+        organizationId = get(originalInput, 'organization.connect.id')
+    }
+    if ( operation === 'update') {
+        if (!itemId) throw new Error('no itemId')
```

Comment:
why not just return false? `if (!itemId) return false`

Reply:
Yes, you can return `false`. It just seemed that the error would make the behavior more predictable. Because the absence of `itemId` when trying to update an entity, returning `false` is more like a lack of access and not an error